.changeset/late-toes-develop.md

```diff
@@ -0,0 +1,11 @@
+---
+"@chakra-ui/button": major
+"@chakra-ui/layout": major
+"@chakra-ui/modal": major
+"@chakra-ui/styled-system": major
+"@chakra-ui/tag": major
```

Comment:
Since these are not breaking changes, I'd recommend changing the versions to `minor` instead of `major`.

Reply:
I changed , please re-check